portfolio_D.css

```diff
@@ -0,0 +1,408 @@
+@media only screen and (min-width: 33rem) {
+  body {
+    max-width: 1440px;
+    margin: auto;
+    padding: 0;
+    background: #e6e2e2;
+    font-family: "Poppins", sans-serif;
+  }
```

Comment:
- [ ] Please notice that you desktop design has affected your mobile design. you need to create the styling for your desktop design while keeping the mobile design the way it was previously. please make sure your mobile design is back to matching the Figma design 

### Your Mobile Design Currently
![image](https://user-images.githubusercontent.com/18730106/155621940-d9c147b3-424c-4e8a-a2bf-04e18eed80bc.png)

- [ ] [OPTIONAL] I've noticed you have two CSS files. that may not be necessary. you can just have one CSS file which has all the mobile CSS without any media queries as before. then you add the desktop styling, *BUT* you should do it inside a media query.

Reply:
As requested by the previous reviewer. Please ensure that your desktop design does not affect your mobile design. you need to create the styling for your desktop design while keeping the mobile design the way it was previously.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/155699595-cc2de052-09b7-4fb0-bea5-dad23fc48905.png) | ![image](https://user-images.githubusercontent.com/10332499/155699693-5e0efd09-2729-4528-a2b5-e9e51a242ac6.png)

Reply:
@micheaol  . Nobody is approving the project despite taken effort to implement all changes. Are you people just to find issues or do you have any responsibility to approve or check if changes are implemented?